src/ml/neural_net/CMakeLists.txt

```diff
@@ -83,3 +72,14 @@ make_library(unity_neural_net OBJECT
     unity_coreml_model_export
     ${additional_unity_neural_net_requirements}
 )
+
+make_library(tctensorflow SHARED
+  SOURCES
+  tf_compute_context.cpp
+)
```

Comment:
Do we need to add in the dependence on unity_shared here?  I think that’s what solved it.

Reply:
I never saved it. I have finally made the change. I had also forgotten to make the destructor for tf_compute_context virtual. Done both.